misk/src/main/kotlin/misk/web/interceptors/TracingInterceptor.kt

```diff
@@ -28,9 +29,22 @@ internal class TracingInterceptor internal constructor(private val tracer: Trace
   }
 
   override fun intercept(chain: Chain): Any? {
-    var result: Any? = null
+    val result: Any?
+    var scope: Scope? = null
 
-    tracer.buildSpan("${chain.action}").startActive(true).use { result = chain.proceed(chain.args) }
+    try {
+      scope = tracer.buildSpan("${chain.action}").startActive(true)
+      result = chain.proceed(chain.args)
+      if (result is Response<*> && result.statusCode / 100 > 3) {
+        Tags.HTTP_STATUS.set(scope.span(), result.statusCode)
+        Tags.ERROR.set(scope.span(), true)
+      }
+    } catch (exception: Exception) {
```

Comment:
I think you want to make sure this is installed to run after the`ExceptionHandlingInterceptor`. that way exceptions are properly transformed into responses and you don't have to worry about catching and re-throwing exceptions. if an exception has leaked to here, it's bad and trace marking is probably not our main concern

Reply:
and by after, I really mean before in the chain so it runs after in response processing. closer to the transport layer

Reply:
Correct

Reply:
Ahh, that's my bad. I made that change in the `WebModule` but didn't add it to the PR. (@mmihic pointed out the same thing in the github issue).